xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.h

```diff
@@ -57,6 +57,7 @@ class CDVDInputStreamBluray
   bool Open() override;
   void Close() override;
   int Read(uint8_t* buf, int buf_size) override;
+  int ReadBlocks(uint8_t* buf, int lba, int num_blocks);
```

Comment:
Maybe move this to `private:` section

Reply:
After make ReadBlocks private and squash in one commit should be good for merge.

Reply:
I'm confused, it can't be private.